__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>What is first stage??? Never saw that defined anywhere</p><p> </p><p>Again, dont print these vars manually. Print them in a code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I need to add a better description in README.</p><p>OK, I don't. </p>